### ISSUE TITLE
Support pprof format natively

### ISSUE THREAD

**Issue by tomershafir** [grafana/pyroscope-java#128]
The pyroscope ecosystem is around Go pprof afaik. I think it makes sense that the agent would be able to emit pprof natively to allow writing tools in different languages, as I only see async-profiler java jfr2pprof (which you use in [JPProf](https://github.com/grafana/JPProf)) and your Go server one. For example, I have a node app (unfortunately) where I need to simulate pyroscope api and I would like to receive pprof. I think this should be complementary to an agent capability (to not require the agent) https://github.com/grafana/agent/issues/4716

**Comment by cyriltovena** [comment]
Yes please I think the best we could do is actually send using the new API so all in for doing a pprof conversion.
